deploy/ansible/ballerine_playbook/roles/setup-ballerine/tasks/cleanup-packer-build.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Remove sensitive credential (1)
+  shell: find / -name "authorized_keys" -exec rm -f {} \;
```

Comment:
not fully understand how is it work... does it connect using ssh?

Reply:
So packer works like this 
* creates a t2.micro vm
* installs all the required tools like docker , docker-compose copy the ballerine repo contents for ballerine to work 
* takes a snapshot of the VM
* creates an ami
* because packer created an ec2 instance and performed the above steps there will be few entries that are necessary and we are cleaning them up.
* In second step we add all tools required for ballerine to start when a customer uses the AMI and starts an ec2 instance